pages/_app.js

```diff
@@ -30,31 +31,35 @@ const EditToggle = (isInEditMode) => {
   const { edit, setEdit } = useEditState();
   return (
     <>
-      <button
-        onClick={() => {
-          setEdit(!edit);
-        }}
-        className="editLink"
-      >
-        {edit ? "Exit edit mode" : "Enter edit mode"}
-      </button>
-      <style jsx>{`
-        .editLink {
-          border: none;
-          position: fixed;
-          top: 0;
-          right: 0;
-          background: var(--orange);
-          color: var(--white);
-          padding: 0.5rem 0.75rem;
-          font-weight: bold;
-          text-decoration: none;
-          display: inline-block;
-          border-bottom-left-radius: 0.5rem;
-          cursor: pointer;
-          font-size: 20px;
-        }
-      `}</style>
+      {(Number(process.env.NEXT_PUBLIC_SHOW_EDIT_BTN) || edit) && (
```

Comment:
I understand `NEXT_PUBLIC_SHOW_EDIT_BTN` is a boolean, usage could be documented in the README

Reply:
Pushed an update to the readme.